src/sentry/demo/demo_org_manager.py

```diff
@@ -99,6 +103,21 @@ def create_demo_org(quick=False) -> Organization:
                 delete_organization.apply_async(kwargs={"object_id": org.id})
                 raise
 
+        if settings.DEMO_MOBILE_PROJECTS:
+            with sentry_sdk.start_span(op="handle_mobile_scenario"):
+                try:
+                    handle_mobile_scenario(ios_project, android_project, quick=quick)
```

Comment:
@roggenkemper instead of having two different identical try/except blocks, can we have one big try/except block where we we call both `handle_react_python_scenario` and `handle_mobile_scenario`?

Reply:
yup !